### PR TITLE
Add Cherry Pick Feature to Backport Tool

### PR DESCRIPTION
**Pull request by briandowns**

### CODE REVIEW

File: cmd/backport/main.go [hunk 1]
```diff
@@ -24,12 +25,13 @@ Options:
     -v                   show version and exit
     -r repo              repository that should be used
     -i issue id          original issue id
-    -c commit            commit id that is being backported
+    -c commits           commits is being backported (comma seperated)
```
Comment (brooksn) [inline cmd/backport/main.go:28]:
```suggestion
    -c commits           commits to backport (comma separated)
```

File: cmd/gen_release_report/bin/gen_release_report
Comment (brooksn) [inline cmd/gen_release_report/bin/gen_release_report:1]: Perhaps add bin directories to the gitignore

**Review by galal-hussein — APPROVED** [review]

File: cmd/gen_release_report/bin/gen_release_report
Comment (briandowns) [reply cmd/gen_release_report/bin/gen_release_report:1]: Sorted.

File: repository/repository.go [hunk 2]
```diff
@@ -227,13 +231,15 @@ func CreateBackportIssues(ctx context.Context, client *github.Client, origIssue
...
 type PerformBackportOpts struct {
-	Repo     string `json:"repo"`
-	CommitID string `json:"commit_id"`
-	IssueID  uint   `json:"issue_id"`
-	Branches string `json:"branches"`
+	Repo           string   `json:"repo"`
+	Commits        []string `json:"commits"`
+	IssueID        uint     `json:"issue_id"`
+	Branches       string   `json:"branches"`
+	PrivateKeyFile string   `json:"private_key_file"` // full path
```
Comment (brooksn) [inline repository/repository.go:238]: Is PrivateKeyFile used?

**Review by brooksn — APPROVED** [review]

File: repository/repository.go (hunk 2 above)
Comment (briandowns) [reply repository/repository.go]: Removed.

**Pull request merged**